app/helpers/manage_submissions_helper.rb

```diff
@@ -8,4 +8,8 @@ def eligible_for_evaluation?(submission)
   def selected_to_advance?(submission)
     submission.judging_status.in?(%w[winner])
   end
+
+  def phase_has_recused_evaluator?(phase)
+    EvaluatorSubmissionAssignment.where(submission: phase.submissions).exists?(status: :recused)
```

Comment:
there is a builtin scope for the recused status, by virtue of the `status` being enum. maybe this could also use the new Phase association you added?
```suggestion
    phase.evaluator_submission_assignments.recused.exists?
```